spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -91,6 +91,32 @@ def paginated_collection(*args)
       end
     end
 
+    context "when specifying download_url_options" do
+      let(:view) do
+        view = mock_action_view
+        allow(view.request).to receive(:query_parameters).and_return details: '1'
+        allow(view.request).to receive(:path_parameters).and_return  controller: 'admin/users', action: 'show', id: 1
+        view
+      end
+
+      let(:collection) do
+        posts = 10.times.map{ Post.new }
+        Kaminari.paginate_array(posts).page(1).per(5)
+      end
+
+      let(:pagination) { paginated_collection(collection,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.